tests/unit/CorrectBusinessType.spec.ts

```diff
@@ -97,6 +97,10 @@ describe('CorrectBusinessType in an Alteration', () => {
   })
 
   it('displays the confirm articles checkbox and enables Done btn when selected', async () => {
+    // Set the selected entity to Benefit Company
+    wrapper.vm.selectedEntityType = 'BEN'
```

Comment:
Hmm, normally we don't set VM properties this way.

Maybe use `wrapper.setData()` ?

https://vue-test-utils.vuejs.org/api/wrapper/setdata.html

Reply:
And you might not need the `nextTick()`.

Reply:
We've done this lots!
Can try, but i think it's more or less the same thing, setting a property local to the component.

Reply:
Updated!